docs/installguide/system_requirements.rst

```diff
@@ -1,12 +1,87 @@
 System requirements
 ===================
 
+Operating systems
+-----------------
+
+ * Windows XP, Vista, 7, 8, 10
```

Comment:
If moved it to the list "limited support", let's just assume it works okay, but if people really insist to use it, we might tell them to download older versions of KA Lite in case they encounter problems.